TOMM20.py

```diff
@@ -0,0 +1,263 @@
+        return RNASequence(output_seq)
+
+
+class RNASequence(NucleicAcidSequence):
+    """
+    RNA sequence.
+    Attributes:
+    -   seq (str): nucleic acid sequence
+    -   seq_type (str): type of nucleic acid sequence (DNA or RNA)
+    -   complement_alphabet (dict): dictionary with complement pairs for DNA or RNA
+    """
+
+    def __init__(self, seq):
+        super().__init__(seq)
+        super().type_definition()
+        if not super().check_seq_type('RNA'):
+            raise ValueError(f'Sequence {self.seq} is not RNA!')
+
+    def transcribe_reverse(self):
+        """
+        Transcribe and then reverse RNA sequence to DNA
+        return:
+        -   DNA sequence
+        """
+        output_seq = self.seq.replace('U', 'T').replace('u', 't')
+        return DNASequence(output_seq[::-1])
+
+
+class AminoAcidSequence(BiologicalSequence):
+    """
+    Protein (amino acid) sequence.
+    Attributes:
+    -   seq (str): protein sequence
+    -   seq_type (str): The type of molecule in the sequence
+    """
+
+    def __init__(self, seq):
+        super().__init__(seq)
+        super().type_definition()
+        if not super().check_seq_type('Protein'):
+            raise ValueError(f'Sequence {self.seq} is not protein')
+
+    def counting_molecular_weight(self):
+        """
+        Counts the molecular mass of a protein sequence seq
+        Arguments:
+        - seq (str): sequence to count the molecular weight
+        Return:
+        - output (int): molecular weight value
+        """
+        output = 0
+        for amino_acid in self.seq:
+            output += DICT_MOLECULAR_MASS[amino_acid]
+        return output - 18 * (len(self.seq) - 1)
+
+def filter_fastq(input_path: str, gc_bounds: tuple or int = (0, 100),
+                 length_bounds: tuple or int = (0, 2 ** 32),
+                 quality_threshold: int = 0,
+                 output_filename=None,
+                 output_data_dir: str = 'filter_fastq_results'):
+    """
+    Main function that is used to get fastq sequence(s) and command. It performs a given action
+    :param input_path: given path where input file is
+    :param gc_bounds: GC composition interval (in percent) for filtering (default is (0, 100)
+    :param length_bounds: length interval for filtering (default is (0, 2 ** 32)
+    :param quality_threshold: threshold value of average quality of reid for filtering, default value is 0
+    :param output_filename: name file with filtered sequences will be saved
+    :param output_data_dir: path where the file with filtered sequences will be saved
+    """
+    if not os.path.isdir(output_data_dir):
+        os.mkdir(output_data_dir)
+    if output_filename is None:
+        output_filename = 'filtered_fastq'
+
+    with open(input_path) as handle, open(os.path.join(output_data_dir, output_filename), mode='w') as file:
+        record = SeqIO.parse(handle, "fastq")
+        for lin in record:
```

Comment:
Скорее придирка, но такие названия кажутся более логичными
```suggestion
        records = SeqIO.parse(handle, "fastq")
        for record in records:
```